scripts/update_specs_from_pricing.py

```diff
@@ -41,9 +41,11 @@
     'South America (Sao Paulo)': 'sa-east-1',
     'US East (N. Virginia)': 'us-east-1',
     'US East (Ohio)': 'us-east-2',
+    'US East (Verizon) - Boston': 'us-east-1',
     'US West (N. California)': 'us-west-1',
     'US West (Oregon)': 'us-west-2',
     'US West (Los Angeles)': 'us-west-2',
+    'US West (Verizon) - San Francisco Bay Area': 'us-west-2',
```

Comment:
https://aws.amazon.com/blogs/industries/verizon-and-aws-deliver-mobile-edge-computing-to-customers-in-boston-and-the-bay-area/